reference/promise-types/reports.markdown

```diff
@@ -6,19 +6,18 @@ tags: [reference, bundle common, reports, promises]
 ---
 
 Reports promises simply print messages. Outputting a message without
-qualification can be a dangerous operation. In a large installation it could
-unleash an avalanche of messaging.
+qualification can be a dangerous operation. In a large installation it
+could unleash an avalanche of messaging, so it is reccomended that reports are
+appropriately.
 
-```cf3
-    reports:
+[%CFEngine_include_example(reports.cf)%]
 
-    "literal string or file refererence",
-       printfile = printfile_body,
-       ...;
-```
+{% comment %} TODO: Should link to a page that describes all cfengine output
+here where we explain that output from reports type promises are
+prefixed with the letter R. {% endcomment %}
```

Comment:
Didn't know about this notation. This will cause it not to appear in the documentation rendering?